zipline/data/us_equity_loader.py

```diff
@@ -250,7 +251,7 @@ def _ensure_sliding_windows(self, assets, dts, field):
                     adjs = {}
                 window = Float64Window(
                     array[:, i].reshape(prefetch_len, 1),
-                    dtype_,
+                    view_kwargs,
```

Comment:
The relevant corresponding change is https://github.com/quantopian/zipline/pull/1174/files#diff-9c48be544e3a72c4ea00f841fe7f9b7bL96.

Previously, the low-level array iterator would do:

```
self._data.view(self.viewtype)
```
it now instead does:
```
self._date.view(**self.view_kwargs)
```

The difference is that we used to always pass a dtype to `view`, but we now pass an empty dict (which is a no-op).  The alternative would have been to pass `None` or another sentinel and have `AdjustedArrayWindow` check its viewtype on every iteration, but it makes the downstream logic cleaner to just always receive a dictionary.